chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,437 @@
 {
-    "info": {
-        "name": "certified-product-controller",
-        "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
-    },
-    "item": [
-        {
-            "name": "GET /rest/certified_products/{listingId} - schema tests",
-            "event": [
-                {
-                    "listen": "test",
-                    "script": {
-                        "exec": [
-                            "const resp = pm.response.json()\r",
-                            "pm.test(\"expect Listing Basic response to have svapResultUrl\", () => {\r",
-                            "  pm.expect(resp).to.have.property('svapNoticeUrl');\r",
-                            "});"
-                        ],
-                        "type": "text/javascript"
-                    }
-                }
-            ],
-            "request": {
-                "method": "GET",
-                "header": [
-                    {
-                        "key": "API-Key",
-                        "value": "{{apiKey}}",
-                        "type": "text"
-                    },
-                    {
-                        "key": "Content-Type",
-                        "value": "application/json",
-                        "type": "text"
-                    }
-                ],
-                "url": {
-                    "raw": "{{url}}/rest/certified_products/10457",
-                    "host": [
-                        "{{url}}"
-                    ],
-                    "path": [
-                        "rest",
-                        "certified_products",
-                        "10457"
-                    ]
-                },
-                "description": "Expect Listing Basic response to have svapResultsUrl"
-            },
-            "response": []
-        }
-    ],
-    "protocolProfileBehavior": {}
+	"info": {
+		"_postman_id": "8b7e235b-2704-42b7-b3ed-42fef144e70c",
+		"name": "certified-product-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET : certified_product: data driven tests",
+			"item": [
+				{
+					"name": "GET /certified_products- 2015 listings",
```

Comment:
We should not combine test by data file used for the requests. You can combine test data used in this 2015 listings folder and below 2014/2011 listings together. You do not need to add separate folder.

Reply:
Combined test data files for all edition of listings and removed the separate folder